src/ResourceManager/Profile/Commands.Profile.Test/Commands.Profile.Test.Netcore.csproj

```diff
@@ -1,12 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <Import Project="..\..\..\..\tools\Common.Netcore.Dependencies.targets" />
-
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <AssemblyName>Microsoft.Azure.Commands.Profile.Test</AssemblyName>
     <RootNamespace>Microsoft.Azure.Commands.Profile.Test</RootNamespace>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <RuntimeIdentifiers>win10-x64;linux-x64;osx.10.12-x64</RuntimeIdentifiers>
```

Comment:
[RuntimeIdentifiers](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#runtimeidentifiers) are only used for [publishing](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-publish?tabs=netcore2x), and should not be required.